### PR TITLE
Fix grammar in Forms Basic lesson

### PR DESCRIPTION
**Pull request by eltonbautista**
- [X] You have read [The Odin Projects Contributing Guide](https://github.com/TheOdinProject/curriculum/blob/main/CONTRIBUTING.md).
 - [X] The title summarizes the change and where it happened, for example: "Fixes punctuation in Clean Code lesson".
 - [X] If the PR is related to an open issue, use a [relevant keyword](https://docs.github.com/en/github/writing-on-github/working-with-advanced-formatting/using-keywords-in-issues-and-pull-requests) and reference it with the `#` sign and the issue number.
 - [X] You have previewed your markdown (if any) in the [Odin Markdown Preview Tool](https://www.theodinproject.com/lessons/preview)
 - [X] If changes are requested, please make the changes in a timely manner. After you submit the changes, request another review from the maintainer (top right).

#### 1. Describe the changes made and include why they are necessary or important:

Extra 'a' on line 417 causes a grammar issue. 

#### 2. Related Issue

Closes #XXXXX

### CODE REVIEW

**Comment by zachmmeyer** [comment]
If you can resolve this conflict, it can be approved. I actually like this change. We did have a recent fix, but this one seems to be more correct.

**Comment by eltonbautista** [comment]
I think I've resolved the conflict. I've never done it before so I'm not sure if I did it correctly though. I tried resolving it from my terminal, but it showed that there were no conflicts. I just resolved the conflict in the widget found on this PR.

File: html_css/intermediate_html/form-basics.md [hunk 1]
```diff
@@ -415,7 +415,8 @@ To create a fieldset, we use the `<fieldset>` element. Whatever form inputs we w
 
 **Legend**
 
-The legend element is used to give a fieldset a heading or caption so the user can see what a grouping of inputs is for.
+
```
Comment (zachmmeyer) [inline html_css/intermediate_html/form-basics.md:418]: Looks like a line was accidentally added when you fixed the conflict. If you could take that out, we'll be done!

```suggestion
```

**Review by zachmmeyer — CHANGES_REQUESTED** [review]
One minor change!

**Review by eltonbautista — COMMENTED** [review]
Oh sorry about that I didn't even notice. I think I've gone ahead and made the suggested change. Thanks!

**Review by zachmmeyer — APPROVED** [review]
Thank you!

**Pull request merged**